src/Components/AddTask/AddTask.tsx

```diff
@@ -0,0 +1,175 @@
+              <span className="text-danger ">title is required</span>
+            )}
+          </div>
+          <div className="form-group my-3">
+            <label className="label-title mb-2">Description</label>
+            <textarea
+              {...register("description", {
+                required: true,
+              })}
+              rows={5}
+              type="text"
+              name="description"
+              className="form-control"
+              placeholder="Enter description..."
+            ></textarea>
+
+            {errors.title && errors.title.type === "required" && (
+              <span className="text-danger ">title is required</span>
+            )}
+          </div>
+          <div className="row">
+            <div className="col-md-6">
+                <select
+                {...register("employeeId", { required: true, valueAsNumber: true })}
+                aria-label="Default select example"
+                type="number"
+                className="form-select"
+              >
+                <option value="" className="text-muted">
+                  User
+                </option>
+                <option >name</option>
+                {/* {
+                    tagsList?.map((tag) =>(
+                      <>
+                      <option key={tag.id} value={tag.id}>{tag.name}</option>
+                      </>
+                    ))
+                  } */}
+              </select>
+                {errors.employeeId && errors.employeeId.type === "required" && (
+                  <span className="text-danger ">No User Selected</span>
+                )}
+            </div>
+            <div className="col-md-6 ">
+                  <select
+                  {...register("projectId", { required: true, valueAsNumber: true })}
+                  aria-label="Default select example"
+                  type="number"
+                  className="form-select mt-sm-3 mt-md-0"
+                >
+                  <option value="" className="text-muted">
+                    Project
+                  </option>
+
+                  {
+                       projects?.map((project:projectType) =>(
+                        <>
+                        <option key={project?.id} value={project.id}>{project?.title}</option>
+                        </>
+                      ))
+                    }
+                </select>
+                {errors.projectId && errors.projectId.type === "required" && (
+                  <span className="text-danger ">No Status Selected</span>
+                )}
+            </div>
+          </div>
+
+          <div className="form-group my-3 d-flex justify-content-between align-items-center">
+            <button
+            onClick={goBack}
+             className="btn btn-outline-danger rounded-5">
+              Cancel
+            </button>
+            <button
+              className={"btn btn-orange my-3 px-4" + (isLoading ? " disabled" : "")}
```

Comment:
you can make it as:
disabled={isLoading}
this is easier to read, the button is disabled whenever the isLoading state is true.